Detectors/TRD/simulation/include/TRDSimulation/Digitizer.h

```diff
@@ -83,25 +83,43 @@ class Digitizer
   int mEventID = 0;
   int mSrcID = 0;
 
-  bool mSDigits{false};                                    // true: convert signals to summable digits, false by defaults
-  std::vector<HitType> mHitContainer;                      // the container of hits in a given detector
-  std::vector<MCLabel> mMergedLabels;                      // temporary label container
-  std::array<SignalContainer, kNdet> signalsMapCollection; // container for caching signals over a timeframe
-  std::array<DigitContainer, kNdet> digitsCollection;      // container for caching digits for paralellization
+  // Digitization parameters
+  static constexpr float mAmWidth = TRDGeometry::amThick();  // Width of the amplification region
+  static constexpr float mDrWidth = TRDGeometry::drThick();  // Width of the drift retion
+  static constexpr float mDrMin = -0.5 * mAmWidth;           // Drift + Amplification region
+  static constexpr float mDrMax = mDrWidth + 0.5 * mAmWidth; // Drift + Amplification region
```

Comment:
just one more thing: constexpr variables are not ordinary members and they should not be prefixed with m.
See here: https://github.com/AliceO2Group/CodingGuidelines/blob/master/naming_formatting.html

Such variables are simply CamelCase.